dm/checker/check_test.go

```diff
@@ -118,28 +123,26 @@ func (s *testCheckerSuite) TestVersionChecking(c *tc.C) {
 		},
 	}
 
-	mock := conn.InitMockDB(c)
+	mock := initMockDB(c)
 	mock.ExpectQuery("SHOW GLOBAL VARIABLES LIKE 'version'").WillReturnRows(sqlmock.NewRows([]string{"Variable_name", "Value"}).
 		AddRow("version", "5.7.26-log"))
 	c.Assert(CheckSyncConfig(context.Background(), cfgs, common.DefaultErrorCnt, common.DefaultWarnCnt), tc.IsNil)
 
-	mock = conn.InitMockDB(c)
-
+	mock = initMockDB(c)
 	mock.ExpectQuery("SHOW GLOBAL VARIABLES LIKE 'version'").WillReturnRows(sqlmock.NewRows([]string{"Variable_name", "Value"}).
 		AddRow("version", "10.1.29-MariaDB"))
 	c.Assert(CheckSyncConfig(context.Background(), cfgs, common.DefaultErrorCnt, common.DefaultWarnCnt), tc.IsNil)
 
-	mock = conn.InitMockDB(c)
+	// TODO: now can't return warning from checker.
```

Comment:
seems that the test shows when ther're only warnings and no errors, nothing is returned. So will user knows there're warnings through dmctl? (is there an integration test to check that?)

Reply:
No place can show only warnings.

[there](https://github.com/pingcap/tiflow/pull/3812) are some content to solve it, but I will open another pr tonight for convenient review.

According to our discussion, I will show it when `check-task/start-task/resume-task` return.